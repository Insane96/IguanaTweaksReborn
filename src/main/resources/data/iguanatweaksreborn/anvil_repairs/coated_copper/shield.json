{
	"item_to_repair": {
		"id": "iguanatweaksreborn:coated_copper_shield"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:obsidian"
			},
			"amount": 4
		}
	]
}