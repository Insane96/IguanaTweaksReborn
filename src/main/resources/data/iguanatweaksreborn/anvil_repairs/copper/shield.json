{
	"item_to_repair": {
		"id": "iguanatweaksreborn:copper_shield"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:copper_ingot"
			},
			"amount": 4
		}
	]
}