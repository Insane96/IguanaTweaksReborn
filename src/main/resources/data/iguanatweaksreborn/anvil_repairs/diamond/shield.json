{
	"item_to_repair": {
		"id": "shieldsplus:diamond_shield"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 3
		}
	]
}