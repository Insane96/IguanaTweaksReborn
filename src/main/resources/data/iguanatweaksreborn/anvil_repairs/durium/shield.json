{
	"item_to_repair": {
		"id": "shieldsplus:durium_shield"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 4,
			"max_repair": 0.5
		},
		{
			"repair_material": {
				"id": "iguanatweaksreborn:durium_ingot"
			},
			"amount": 2
		},
		{
			"repair_material": {
				"id": "iguanatweaksreborn:durium_nugget"
			},
			"amount": 18
		}
	]
}