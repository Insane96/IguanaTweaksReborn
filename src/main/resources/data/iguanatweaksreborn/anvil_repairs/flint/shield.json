{
	"item_to_repair": {
		"id": "iguanatweaksreborn:flint_shield"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:flint"
			},
			"amount": 4
		}
	]
}