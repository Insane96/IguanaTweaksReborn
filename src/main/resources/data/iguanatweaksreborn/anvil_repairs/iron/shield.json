{
	"item_to_repair": {
		"id": "shieldsplus:iron_shield"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 4
		},
		{
			"repair_material": {
				"id": "minecraft:iron_nugget"
			},
			"amount": 36,
			"cost_multiplier": 9
		}
	]
}