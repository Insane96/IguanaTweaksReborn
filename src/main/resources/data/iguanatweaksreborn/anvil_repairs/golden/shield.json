{
	"item_to_repair": {
		"id": "shieldsplus:golden_shield"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:gold_ingot"
			},
			"amount": 4
		},
		{
			"repair_material": {
				"id": "minecraft:gold_nugget"
			},
			"amount": 36
		}
	]
}